changelog/README.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
+
 ## 5th February 2025
 * Extended [list of supported filters](../guidelines/filtering.md).
+* Fixed [list of supported filters on invoices page](../operations/invoices.md)
```

Comment:
I prefer Changelog entries that are specific with what the change is, and also puts the affected API operation first. BTW I am planning to create a Changelog style guide for all Mews APIs, to make it easier for developers to create changelog entries.
```suggestion
* [Get invoices](../operations/invoices.md#get-invoices): Extended Filters to include `registerIdEq`.
```